_includes/components/2-col.html

```diff
@@ -1,4 +1,4 @@
-<article class="container {{ include.class }} page-content__prose">
```

Comment:
For consistency and improved screenreader use, all instances of `<article>` in this repo have been replaced with a `<div>`